src/lib/database/seed.ts

```diff
@@ -1,18 +1,64 @@
+import { seam_logo } from "lib/seam_logo.ts"
+
 import type { Database } from "./schema.ts"
 
-export interface Seed {
-  thing_1: string
-  api_key: string
-}
+export type Seed = ReturnType<typeof seedDatabase>
```

Comment:
I think having `Seed` defined here vs return type was intentional to make it easy for people to find the seed structure. If we really want to adopt this, let's PR it into the template.

Reply:
🤷 imo some deviation from the template is ok, personally I prefer this style because I'm lazy 😛